src/Compilers/CSharp/Portable/Symbols/FieldSymbol.cs

```diff
@@ -382,11 +382,11 @@ public sealed override bool HasUnsupportedMetadata
         /// <summary>
         /// Is this a field of a tuple type?
         /// </summary>
-        public virtual bool IsTupleField
+        public bool IsTupleField
         {
             get
             {
-                return false;
+                return ContainingType.IsTupleType;
```

Comment:
>ContainingType.IsTupleType [](start = 23, length = 26)

How is the `IsTupleField` property used? It seems potentially useful to know if a field is a tuple element but less useful to know that it is simply a field of a tuple.